src/main/java/com/englishtown/vertx/elasticsearch/impl/DefaultElasticSearchService.java

```diff
@@ -244,7 +245,10 @@ public void search(List<String> indices, SearchOptions options, Handler<AsyncRes
             if (!options.getSorts().isEmpty()) {
                 options.getSorts().forEach(sort -> builder.addSort(sort.getField(), sort.getOrder()));
             }
-            if (options.getExtraSource() != null) builder.setExtraSource(options.getExtraSource().encode());
```

Comment:
Yeah this must be a mistake, was pretty sure I fixed that during the conflict though, I'll resubmit.